src/Maestro/Maestro.Data/Migrations/20191014170741_AddDateAddedToBuildChannels.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace Maestro.Data.Migrations
+{
+    public partial class AddDateAddedToBuildChannels : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<DateTimeOffset>(
+                name: "DateAdded",
+                table: "BuildChannels",
+                nullable: false,
+                defaultValue: new DateTimeOffset(new DateTime(1, 1, 1, 0, 0, 0, 0, DateTimeKind.Unspecified), new TimeSpan(0, 0, 0, 0, 0)));
+        }
+
+        protected override void Down(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropColumn(
+                name: "DateAdded",
```

Comment:
`DateTimeAdded` would be more accurate.  Maybe make this `ModifiedDateTime`?

Reply:
I don't know if Modified really makes sense. I chose added because the time represents the time that a build is added to a channel. I can change to DateTimeAdded, though I just went with the convention that we used in Builds, where we have DateProduced, which is a DateTimeOffset as well.

Reply:
DateTimeAdded sounds fine